README.md

```diff
@@ -366,6 +366,31 @@ const connector = new Connector({
 });
 ```
 
+This can be useful when configuring credentials that differ from
+Application Default Credentials.
+
+The below example showcases how to initialize a `Connector` from a service account key that is
+[loaded into an environment variable][google-auth-load-creds].
+
+```js
+import {Connector} from '@google-cloud/cloud-sql-connector';
+import {GoogleAuth} from 'google-auth-library';
+
+// load the environment variable with our keys
+const keysEnvVar = process.env.CREDS
+if (!keysEnvVar) {
+  throw new Error('The $CREDS environment variable was not found!');
+}
+const keys = JSON.parse(keysEnvVar);
+
+const connector = new Connector({
```

Comment:
This is a nice addition, however I'm a bit concerned with customers assuming this is the recommended way to setup their auth flow. I would recommend pointing to either of the following with a helpful description:
- https://github.com/googleapis/google-auth-library-nodejs/
- https://github.com/googleapis/google-auth-library-nodejs/tree/main/samples

Reply:
I see https://github.com/googleapis/google-auth-library-nodejs/#loading-credentials-from-environment-variables which has a nice explanation of credential usage, but it's not end-to-end and still leaves a lot for customers to figure out. So how about we lead with:

"This can be useful when configuring credentials that differ from
Application Default Credentials. See the [documentation](https://github.com/googleapis/google-auth-library-nodejs/#loading-credentials-from-environment-variables) on google-auth for more information." or similar.

Reply:
Sound good i have removed the specific example in lieu of linking to docs.